.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,15 @@
+name: Trivy
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "30 20 * * *"
+
+jobs:
+  call_workflow:
+    name: Run Trivy Scan Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/trivy-scan-template.yml@main
+    secrets: inherit
+    with:
+      additional-build-flags: "-x :twilio-examples:build"
```

Comment:
Shall we add the EOF line?

Reply:
Changed with [1def7d5](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/1def7d57b9d739e97a12749d9bffcf257fa19b52).